src/EnergyPlus/HeatPumpWaterToWaterSimple.cc

```diff
@@ -1737,7 +1737,7 @@ void GshpSpecs::CalcWatertoWaterHPCooling(EnergyPlusData &state, Real64 const My
 
     // SUBROUTINE PARAMETER DEFINITIONS:
     Real64 const CelsiustoKelvin(DataGlobalConstants::KelvinConv); // Conversion from Celsius to Kelvin
```

Comment:
Why not this one?

Reply:
Because my script wasn't smart enough to find it. I'll fix this one up.